### PR TITLE
South Carolina EITC

### PR DESCRIPTION
**Pull request by hua7450**

### CODE REVIEW

File: policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  2022-01-01: 1.0417
+  2021-01-01: 0.8333
+  2020-01-01: 0.625
+  2019-01-01: 0.4167
+  2018-01-01: 0.2083
+metadata:
+  unit: year
+  label: South Carolina EITC Rate
+  reference:
+    title: South Carlina Earned income tax credit
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml:12]:
```suggestion
   -  title: South Carlina Earned income tax credit 
```

File: policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+description: South Carolina provides this rate to match the federal EITC.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml:1]:
```suggestion
description: South Carolina matches the federal EITC at this rate.
```

File: policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+values:
+  2022-01-01: 1.0417
+  2021-01-01: 0.8333
+  2020-01-01: 0.625
+  2019-01-01: 0.4167
+  2018-01-01: 0.2083
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml:7]: start with oldest value at the top

File: policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  unit: year
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml:9]:
```suggestion
  unit: /1
```

File: policyengine_us/tests/policy/baseline/gov/states/sc/tax/income/credits/eitc/sc_eitc.yaml
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/sc/tax/income/credits/eitc/sc_eitc.yaml:1]: need unique test case names to represent simulated situation

File: policyengine_us/variables/gov/states/sc/tax/income/credits/eitc/sc_eitc.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+import numpy as np
+
+
+class sc_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "South Carolina EITC"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://dor.sc.gov/about/forms"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/sc/tax/income/credits/eitc/sc_eitc.py:11]: need exact form reference

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]
comments

File: policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+  2018-01-01: 0.2083
+  2019-01-01: 0.4167
+  2020-01-01: 0.625
+  2021-01-01: 0.8333
+  2022-01-01: 1.0417
+metadata:
+  unit: /1
+  label: South Carolina EITC Rate
+  reference:
+  - title: South Carlina Earned income tax credit
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml:12]: need 2022 reference

Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml:12]:
```suggestion
  - title: South Carlina Schedule TC-60 2021
```

File: policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  2018-01-01: 0.2083
+  2019-01-01: 0.4167
+  2020-01-01: 0.625
+  2021-01-01: 0.8333
+  2022-01-01: 1.0417
+metadata:
+  unit: /1
+  label: South Carolina EITC Rate
+  reference:
+  - title: South Carlina Schedule TC-60 2021 (2021)
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml:12]:
```suggestion
  - title: South Carolina Schedule TC-60 2021 (2021)
```

File: policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+values:
+  2018-01-01: 0.2083
+  2019-01-01: 0.4167
+  2020-01-01: 0.625
+  2021-01-01: 0.8333
+  2022-01-01: 1.0417
+metadata:
+  unit: /1
+  label: South Carolina EITC Rate
+  reference:
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml:11]: Can you add a legal code reference?

File: policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml [hunk 9]
```diff
@@ -0,0 +1,16 @@
+description: South Carolina matches the federal EITC at this rate.
+values:
+  2018-01-01: 0.2083
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml:3]: These values need references right? Legal code may suffice.

File: policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+  2019-01-01: 0.4167
+  2020-01-01: 0.625
+  2021-01-01: 0.8333
+  2022-01-01: 1.0417
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/sc/tax/income/credits/eitc/rate.yaml:7]: Per law
```suggestion
  2022-01-01: 1.0417
  2023-01-01: 1.25
```

**Review by PavelMakarchuk — COMMENTED** [review]
need to run `git pull upstream master`

File: policyengine_us/variables/gov/states/sc/tax/income/credits/eitc/sc_eitc.py [hunk 11]
```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+import numpy as np
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/sc/tax/income/credits/eitc/sc_eitc.py:2]: The `model_api` import includes `np`
```suggestion
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**